src/osd/OSD.cc

```diff
@@ -5846,10 +5846,20 @@ void OSD::handle_scrub(MOSDScrub *m)
       PG *pg = p->second;
       pg->lock();
       if (pg->is_primary()) {
+        if (m->stop_scrub && !pg->scrubber.must_scrub) {
+	  derr << __func__ << *pg << " stop_scrub should stop manual scrub" << dendl;
```

Comment:
It looks to me like this message will happen even if no scrubbing at all is going on.  scrubber.active could be false.